src/v2/ion/RemediationConstants.js

```diff
@@ -41,6 +41,7 @@ const FORMS = {
   DEVICE_APPLE_SSO_EXTENSION: 'device-apple-sso-extension',
   CANCEL_TRANSACTION: 'cancel-transaction',
   LAUNCH_AUTHENTICATOR: 'launch-authenticator',
+  DEVICE_ENROLLMENT_TERMINAL: 'device-enrollment',
```

Comment:
rename the actual value to `device-enrollment-terminal` as well.

Reply:
Done